agent/errors.go

```diff
@@ -13,10 +13,5 @@ var (
 	errAgentStarted    = errors.New("agent: already started")
 	errAgentNotStarted = errors.New("agent: not started")
 
-	errTaskNoController         = errors.New("agent: no task controller")
-	errTaskNotAssigned          = errors.New("agent: task not assigned")
-	errTaskStatusUpdateNoChange = errors.New("agent: no change in task status")
-	errTaskUnknown              = errors.New("agent: task unknown")
-
-	errTaskInvalid = errors.New("task: invalid")
```

Comment:
All of these were unused?

Reply:
Yes, as listed in the changelog message from the commit:

> agent/errors.go:7:1￼ errTaskNoController is unused (deadcode)
> agent/errors.go:7:1￼ errTaskStatusUpdateNoChange is unused (deadcode)
> agent/errors.go:7:1￼ errTaskNotAssigned is unused (deadcode)
> agent/errors.go:7:1￼ errTaskInvalid is unused (deadcode)

Reply:
Note that these are private to the package so if it compiles those can safely be removed

Reply:
Sorry, had missed that.